### PR TITLE
thrift: delete thrift

### PR DESCRIPTION
**Pull request by dapengzhang0**

### CODE REVIEW

**Comment by ejona86** [comment]
We can just delete it.

**Comment by dapengzhang0** [comment]
Deleted.

File: all/build.gradle [hunk 1]
```diff
@@ -20,8 +20,7 @@ def subprojects = [
   project(':grpc-protobuf'),
   project(':grpc-protobuf-lite'),
   project(':grpc-protobuf-nano'),
-  project(':grpc-stub'),
```
Comment (ejona86) [inline all/build.gradle]: Please leave the trailing comma.

**Comment by dapengzhang0** [comment]
retest this please

**Comment by dapengzhang0** [comment]
I signed it!
Didn't I?

**Comment by ejona86** [comment]
There was talk on the hangout that the cla bot was getting things wrong.

**Comment by dapengzhang0** [comment]
resolved #3477

**Pull request merged**